packages/sdk/tests/docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3.9'
-

```

Comment:
Why?

Reply:
It's depreciated now : 
`WARN[0000] /Users/robin/Documents/DataProtector/dataprotector-sdk/packages/sdk/tests/docker-compose.yml: `version` is obsolete`